src/utils/formatter.py

```
@@ -4,201 +4,216 @@
 def format_help(command_prefix):
     """Format help message.
-    
+
     Args:
         command_prefix: The command prefix used by the bot.
-        
+
     Returns:
         str: Formatted help message.
     """
     help_text = f"**OpenHands Discord Bot Help**\n\n"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)